google/resource_storage_default_object_acl_test.go

```diff
@@ -0,0 +1,213 @@
+package google
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccGoogleStorageDefaultObjectAcl_basic(t *testing.T) {
+	t.Parallel()
+
+	bucketName := testBucketName()
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:     func() { testAccPreCheck(t) },
+		Providers:    testAccProviders,
+		CheckDestroy: testAccGoogleStorageDefaultObjectAclDestroy,
+		Steps: []resource.TestStep{
+			resource.TestStep{
+				Config: testGoogleStorageDefaultObjectsAclBasic1(bucketName),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckGoogleStorageDefaultObjectAcl(bucketName, roleEntityBasic1),
```

Comment:
I think you forgot to copy/paste the variables for `roleEntity...` into the test file?

Reply:
I did it on purpose: i'm using the roleEntity variable definitions from resource_storage_bucket_acl_test.go. This permissions set is  fine to use for testing purposes for all ACL resources. 
Besides, all the tests are passing:
make testacc TEST=./google TESTARGS='-run=TestAccGoogleStorageDefaultObjectAcl'
==> Checking that code complies with gofmt requirements...
TF_ACC=1 go test ./google -v -run=TestAccGoogleStorageDefaultObjectAcl -timeout 120m
=== RUN   TestAccGoogleStorageDefaultObjectAcl_basic
=== RUN   TestAccGoogleStorageDefaultObjectAcl_upgrade
=== RUN   TestAccGoogleStorageDefaultObjectAcl_downgrade
--- PASS: TestAccGoogleStorageDefaultObjectAcl_basic (14.37s)
--- PASS: TestAccGoogleStorageDefaultObjectAcl_upgrade (24.78s)
--- PASS: TestAccGoogleStorageDefaultObjectAcl_downgrade (25.00s)
PASS
ok      github.com/terraform-providers/terraform-provider-google/google 25.240s

Reply:
Oh, right.  Sorry, I was forgetting that those variables would be exposed at the module level.  When I didn't see them in your file, I got very confused. :)